services/QuillLMS/spec/queries/snapshots/period_query_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Snapshots
+  describe PeriodQuery do
+
+    let(:test_period_query) do
+      Class.new(described_class) do
```

Comment:
Cool!  I like this pattern.

Reply:
I originally just declared a new `class` in the test file, but Rubocop has a rule against that.  So this pattern is one I stole from the Rubocop documentation for that rule.  Good job, linter!